src/components/Tutorial/TutorialHint.tsx

```diff
@@ -14,7 +16,15 @@ const TutorialHint: FC<PropsWithChildren> = ({ children }) => {
   return (
     <>
       <a
-        className={`button button-variable-width button-status button-less-padding text-small ${hint ? 'button-dim' : ''}`}
```

Comment:
The TutorialHint has a regression:

## Current Behavior

d3d9f02586

The hint anchor button text is blue.

<img width="753" alt="Brave Browser 2024-08-31 09 18 14" src="https://github.com/user-attachments/assets/53fbeb53-a453-4f20-8e4c-770013e13895">

## Expected Behavior

The hint anchor button text should be black.

<img width="752" alt="Brave Browser 2024-08-31 09 18 09" src="https://github.com/user-attachments/assets/408e4f86-51b3-4ea1-8524-7229fcf0b253">

Reply:
Sorry I missed this